docker-app/qfieldcloud/subscription/models.py

```diff
@@ -794,15 +794,15 @@ def create_subscription(
         account: UserAccount,
         plan: Plan,
         created_by: Person,
-        active_since: Optional[datetime] = None,
-    ) -> tuple[Optional["AbstractSubscription"], "AbstractSubscription"]:
+        active_since: datetime | None = None,
+    ) -> tuple[Type["AbstractSubscription"] | None, "AbstractSubscription"]:
```

Comment:
```suggestion
    ) -> tuple[Type["AbstractSubscription"] | None, "AbstractSubscription"]:
```
I am not familiar with Type. Is it required? tuple["MyClass"] doesn't work?

Reply:
Unfortunately, Python is a bit stupid in ignoring types defined as strings... It tried to run the `|` operators bettween `str` and `None` types instead of ignoring it as type definition, which lead to the following error:
```
  File "/usr/src/app/qfieldcloud/subscription/models.py", line 798, in AbstractSubscription
    ) -> tuple["AbstractSubscription" | None, "AbstractSubscription"]:
TypeError: unsupported operand type(s) for |: 'str' and 'NoneType'
```

I tried so, had to force push to fix that.

Reply:
A good question is whether to use `Type` every time we use string types, or only on demand. I have no strong opionion.

Assuming we want to not use `Optional`, we do have two paths:

Without explicit `Type` it is easier for the developer to type, but changing a type to `Optional` will require to add the `Type` around the string and a bit cryptic error.